npm/qsharp/generate_katas_content.js

```diff
@@ -132,6 +134,15 @@ function resolveSvgSegment(properties, baseFolderPath) {
     `Could not read the contents of the SVG file at ${svgPath}`,
   );
 
+  // An SVG file is basically an HTML file. If it includes blank lines, this will
+  // cause issues when including in Markdown, as blank lines indicate the end of
+  // HTML content. Check for blank lines within the document.
+  if (/\n\r?\n/.test(svg)) {
```

Comment:
This catches the issue that was causing confusion in https://github.com/microsoft/qsharp/issues/1274

Reply:
Should this detect blank lines with whitespace? Not sure if that'd break markdown too

```suggestion
  if (/\n\s*\r?\n/.test(svg)) {
```

Reply:
Good call. Done.